src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -712,6 +715,11 @@ let setup_daemon logger =
         or_from_config json_to_currency_fee_option "snark-worker-fee"
           ~default:Mina_compile_config.default_snark_worker_fee snark_work_fee
       in
+      let node_status_url =
+        maybe_from_config YJ.Util.to_string_option "report-health-url"
```

Comment:
should this have the same name as on the command line, that is, "node-status-url"?